modules/auxiliary/scanner/rdp/rdp_scanner.rb

```diff
@@ -42,7 +42,7 @@ def rdp?
     sock.put(@probe)
     response = sock.get_once(-1)
     if response
-      if RDP_RE.match?(response)
+      if RDP_RE.match(response)
```

Comment:
This works fine in 2.4.1.  It won't work in 2.3 and perhaps earlier versions.  I thought msf was 2.4.x?